pkg/params/settings/config.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 const (
-	ApplicationNameKey                    = "application-name"
-	HubURLKey                             = "hub-url"
-	HubCatalogNameKey                     = "hub-catalog-name"
+	ApplicationNameKey = "application-name"
+	HubURLKey          = "hub-url"
+	HubCatalogNameKey  = "hub-catalog-name"
+	//nolint: gosec
```

Comment:
getting that false positive for that one:

```
pkg/params/settings/config.go:16:2: G101: Potential hardcoded credentials (gosec)
	MaxKeepRunUpperLimitKey               = "max-keep-run-upper-limit
```

🤷🏻

Reply:
yes, saw same